### ISSUE TITLE
add support for ARM64?

### ISSUE THREAD

**Issue by zedd-0** [rcmaehl/MSEdgeRedirect#14]
it would be nice if you could also compile an native ARM64 version.

**Comment by rcmaehl** [comment]
The app should be able to run under the x64 emulation for ARM, but I'm not too familiar with it. Is the app not launching at all under emulation, or is it launching and not doing anything at all?

**Comment by zedd-0** [comment]
it's just a wish/feature-request for the future.
a slim version is always better than an emulation.

today i have not an ARM-system,
but in future, i am interested in one.

**Comment by gnpaone** [comment]
> The app should be able to run under the x64 emulation for ARM, but I'm not too familiar with it. Is the app not launching at all under emulation, or is it launching and not doing anything at all?

No, it doesn't work, emulation means it's like sandboxed, no direct interaction with the computer, native armhf and arm64 versions only will work, for emulating, windows 10 & 11 arm supports x86 & x64. I used Win 10 arm64 (no pics) and armhf versions on Raspberry Pi 4b just to check (u can see that no matter what I search it shows in edge and there is no logs folder too) **(I have a doubt, does this app work on Windows 10?)**
![Screenshot (4870)](https://user-images.githubusercontent.com/78990165/141809913-248eae4c-7116-482f-90e2-c3489b666a24.png)
![Screenshot (4882)](https://user-images.githubusercontent.com/78990165/141809938-a4abaf5d-8091-4df5-a60f-d9c11ed5a778.png)
![Screenshot (4879)](https://user-images.githubusercontent.com/78990165/141809948-99788036-c570-4d2c-bc7f-585322de9741.png)
![Screenshot (4877)](https://user-images.githubusercontent.com/78990165/141809964-64f9b210-a9c6-4b99-baf4-4a760712e1c5.png)
![Screenshot (4876)](https://user-images.githubusercontent.com/78990165/141809980-4c609dc2-c477-467d-a303-edc62768ce49.png)
![Screenshot (4875)](https://user-images.githubusercontent.com/78990165/141809991-93ec0203-aa33-43af-a8f9-06266811871b.png)
![Screenshot (4874)](https://user-images.githubusercontent.com/78990165/141810003-fd0e9da2-56a0-4b92-8ebc-33c46c0b90d3.png)
![Screenshot (4873)](https://user-images.githubusercontent.com/78990165/141810010-1ded4ad5-48eb-4211-849e-d6d4f1ac05e5.png)
![Screenshot (4872)](https://user-images.githubusercontent.com/78990165/141810020-744b7380-ebb4-4996-99db-71c2bebc73f0.png)
![Screenshot (4871)](https://user-images.githubusercontent.com/78990165/141810036-0788cb13-4796-48e0-b4ab-4545b4566506.png)

**Comment by rcmaehl** [comment]
> **(I have a doubt, does this app work on Windows 10?)**

Yes. I'm currently using it on 10. Minimum requirement in Win 8.1

**Comment by CryoRenegade** [comment]
Does AutoIT even have ARM support? Sounds like a pain, especially with Win10 and Win11 ARM in the state it is in.

**Comment by rcmaehl** [comment]
> 

It does not. But I may offer a version in a language that does support ARM. I'm just less experienced in them

**Comment by CryoRenegade** [comment]
Might be easier to rewrite the while thing using a MSVC compatible
language. Rust is also making headlines in windows thanks to cargo.

**Comment by rcmaehl** [comment]
@HarshalKudale I know you're using C#. This may be an option for your solution?

**Comment by HarshalKudale** [comment]
> @HarshalKudale I know you're using C#. This may be an option for your solution?

Yep, someone recently reported issue about NoMoEdge not working on ARM. I am currently looking into solving that along with other bugs. I am currently newbie in c# as well so I will try my best.

**Comment by ylluminate** [comment]
Would be nice to use this with Win11-ARM on Parallels in macOS. Thanks for raising this issue.

**Comment by PeterMaZep** [comment]
this doesn't even work on arm64 because of the sandbox.
